packages/icons/src/library/login.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const login = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path
+			d="M11 14.5l1.1 1.1 3-3 .5-.5-.6-.6-3-3-1 1 1.7 1.7H5v1.5h7.7L11 14.5zM16.8 5h-7c-1.1 0-2 .9-2 2v1.5h1.5V7c0-.3.2-.5.5-.5h7c.3 0 .5.2.5.5v10c0 .3-.2.5-.5.5h-7c-.3 0-.5-.2-.5-.5v-1.5H7.8V17c0 1.1.9 2 2 2h7c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2z"
+			fill-rule="evenodd"
```

Comment:
`fill-rule` and `clip-rule` should be camel case (ex. `fillRule`)

Reply:
Actually try and see if the icon works without them entirely. It's almost always the case. I.e. remove them, see if the icon renders right, if it does, keep them out.

Reply:
Tested without them, works fine so I removed them. Thanks for the pointers! 👍